src/applications/caregivers/tests/unit/components/FormFields/FacilitySearch.unit.spec.js

```diff
@@ -74,7 +74,9 @@ describe('CG <FacilitySearch>', () => {
       loader: container.querySelector('va-loading-indicator'),
       radioList: container.querySelector('va-radio'),
       searchInputError: queryByRole('alert'),
-      moreFacilities: queryByText('Load more facilities'),
+      moreFacilities: container.querySelector(
+        'va-button[text="Load more facilities"]',
```

Comment:
Maybe consider using test IDs instead of querying based on attribute values?

Reply:
So, I really try to avoid using test ids if I can help it. https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change has some good thoughts on why I tend to not use them. I added a helper that can be reused to get a `VaButton` component by the text value that I kind of like how it turned out. Let me know what you think about that. I'm still down to use the test id, but I find using the actual text values to be more resilient to code changes.